python/lsst/ctrl/bps/wms/panda/conf_example/pipelines_check_execution_butler_idf.yaml

```diff
@@ -24,52 +24,30 @@ campaign: quick
 # the default.
 executionButler:
   whenCreate: "SUBMIT"
-  #USER executionButlerDir: "/my/exec/butler/dir"  # if user provided, otherwise uses executionButlerTemplate
-  createCommand: "${CTRL_MPEXEC_DIR}/bin/pipetask qgraph -b {butlerConfig} --input {inCollection} --output-run {outCollection} --save-execution-butler {executionButlerDir} -g {qgraphFile}"
+  createCommand: "${CTRL_MPEXEC_DIR}/bin/pipetask qgraph -b {butlerConfig} -i {inCollection} -o {output} --output-run {outCollection} --save-execution-butler {executionButlerDir} -g {qgraphFile} --skip-existing-in {outCollection}"
+
   whenMerge: "ALWAYS"
   implementation: JOB  # JOB, WORKFLOW
   concurrency_limit: db_limit
 
-  # What commands to run as part of the merge step:
-  command1: "${DAF_BUTLER_DIR}/bin/butler --log-level=VERBOSE transfer-datasets  {executionButlerDir} {butlerConfig} --collections {outCollection}"
-  command2: "${DAF_BUTLER_DIR}/bin/butler collection-chain {butlerConfig} {output} {outCollection} --mode=prepend"
-
-  # For --replace-run behavior need to run two collection-chain commands
-  # in addition to the transfer-datasets:
-  #command2: "${DAF_BUTLER_DIR}/bin/butler collection-chain {butlerConfig} {output} --mode=pop 1"
-  #command3: "${DAF_BUTLER_DIR}/bin/butler collection-chain {butlerConfig} {output} --mode=prepend {outCollection}"
+  mergePreCmdOpts: "{defaultPreCmdOpts}"
+  command1: "${DAF_BUTLER_DIR}/bin/butler {mergePreCmdOpts} transfer-datasets  {executionButlerDir} {butlerConfig} --collections {outCollection}"
+  command2: "${DAF_BUTLER_DIR}/bin/butler {mergePreCmdOpts} collection-chain {butlerConfig} {output} --flatten --mode=extend {inCollection}"
```

Comment:
I thought we were going to use an include file to pick up the execution butler configuration so that we didn't have to write down the commands for panda and separately for condor? Maybe just over-riding defaultPreCmdOpts?

Reply:
needs further discussion